### PR TITLE
browserstack e2e tests

### PR DESCRIPTION
**Pull request by davidmohringmrf**

### CODE REVIEW

File: experiences/homepage/homepage.test.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		expect(firstCardIsInViewport).equal(true);
+
+		const isAtInitialSnapPoint = await isAtSnapPoint(browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.initial);
+
+		expect(isAtInitialSnapPoint).equal(true);
+	});
+
+	// fix minimise
```
Comment (oliverfernandez) [inline experiences/homepage/homepage.test.js:64]: Why is this test commented?

File: experiences/homepage/homepage.test.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	it('activate card by click', async()=>{
+		await touchCard(browser, config.cards.homepage.cardSelector);
+
+		const isAtActiveSnapPoint = await isAtSnapPoint(browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.active);
+
+		expect(isAtActiveSnapPoint).equal(true);
+	});
+});
```
Comment (oliverfernandez) [inline experiences/homepage/homepage.test.js:84]: Based on the [`flowcards.json`](https://github.com/Marfeel/flowcards-playground/blob/9f78fd6dd36d683d087c323e57377eb69b89407f/experiences/homepage/homepage.json#L1) for the homepage experience, I think we have several tests missing:

- `mode: modal`  that is, the blackout foreground appears when the Flowcard is `active` obscuring the content behind

- `button: {action: "setSnapPoint:initial"}` that is, when the Flowcard is open, and you scroll a little bit inside, a close button appears in the upper right corner, and if you click it, the Flowcard goes back to the `initial` snapPoint

- `infiniteScroll: true` that is, when I reach the end of the article, and I try to keep scrolling, the Flowcard starts going up, showing the homepage, in a ""infinite scroll like" experience



For me is ok to land this PR without this test, so we start testing everything together, but please, remember to complete the testing for the whole experience

File: experiences/multicard/multicard.test.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+const { bootstrapExperience } = require("../../tests/utils/bootstrap");
```
Comment (oliverfernandez) [inline experiences/multicard/multicard.test.js]: What's the differenec between `multicard.test.js` and `multicard.localhost.test.js` ? This file seems duplicated?

Comment (davidmohringmrf) [reply experiences/multicard/multicard.test.js]: Its outdated, I'll remove it for now, we are focusing on the homepage experience.

File: experiences/homepage/homepage.test.js (hunk 2 above)
Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:84]: Yes, of course we are going to test the whole experience. This is just the initial POC. 
My idea was to land this on master and then work on additional test features, so we don't overload this PR more.

File: experiences/homepage/homepage.test.js (hunk 1 above)
Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:64]: Its currently not working as expected. Planing to fix it once we go 100% for the homepage experience.

**Review by oliverfernandez — APPROVED** [review]

**Pull request merged**